docs/developer_guide.md

```diff
@@ -259,56 +269,67 @@ Debugging options are available. Click on the required option :
 <details><summary>Use gdb to debug in Docker</summary>
 
 1. Build a project in a debug mode :
-	```
+	```bash
 	make docker_build BAZEL_BUILD_TYPE=dbg
 	```
 
 2. Run the container :
-	```
+	```bash
 	docker run -it --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v ${PWD}:/ovms -p 9178:9178 --entrypoint bash openvino/model_server-build:latest
 	```
-3.	Assuming resnet50 model is prepared for OVMS in /models catalog recompile the OpenVINO Model Server with debug symbols using command:
-    ```
-	[root@72dc3b874772 ovms]# bazel build //src:ovms -c dbg
-    [root@72dc3b874772 ovms]# gdb --args ./bazel-bin/src/ovms --model_name resnet --model_path /models
+3.	Prepare resnet50 model for OVMS in /models catalog and recompile the OpenVINO Model Server in docker container with debug symbols using command:
+	```bash
+	mkdir -p /models/1 && wget -P /models/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin && wget -P /models/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
+	```
+    ```bash
+	bazel build //src:ovms -c dbg
+	```
+	```bash
+    gdb --args ./bazel-bin/src/ovms --model_name resnet --model_path /models
 	```
     > **NOTE**: For best results, use the makefile parameter `BAZEL_BUILD_TYPE=dbg` to build the dependencies in debug mode as shown above
 
 
 - For unit test debugging, run command :
-	```
+	```bash
 	gdb --args ./bazel-bin/src/./ovms_test --gtest_filter='OvmsConfigTest.emptyInput'
 	```
 
-- For forking tests debugging, enable fork follow mode by running command :
+- For forking tests debugging, enable fork follow mode by running command:
 	```
 	# (in gdb cli) set follow-fork-mode child
 	```
 </details>
 
 <details><summary>Use minitrace to display flame graph</summary>
 
+Download the model files and store them in the `models` directory
+```bash
+mkdir -p models/resnet/1
+curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
```

Comment:
Do we have 2022.1 versions of these models?

Reply:
Available. Should I change it also for other files that use it?

Reply:
That would be great.